recipes/xorg-util-macros/bld.bat

```diff
@@ -0,0 +1,7 @@
+:: Just delegate to the Unixy script; but some of the X.org tarballs have
+:: config.guess files too ancient to recognize 64-bit Windows!
+curl -sSL -o config.guess "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess"
+curl -sSL -o config.sub "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub"
```

Comment:
Yes, that's a good idea. This package is actually the perfect place to bundle such files, since its job is exactly to provide miscellaneous, generic X.org build utilities.